.gitignore

```diff
@@ -1,2 +1,16 @@
+# go-llama build artifacts
+go-llama.cpp
+binding.o
+libbinding.a
+
+# llama models
+models
```

Comment:
if we put that in ignore, than it's more commands for the user to type - my plan was to leave a `models` folder with a `.keep` file checked out, so you can just copy the models in